scripts/gha/integration_testing/xcodebuild.py

```diff
@@ -46,6 +46,7 @@ def get_args_for_build(path, scheme, output_dir, ios_sdk, configuration):
     output_dir (str): Directory for the resulting build artifacts. Will be
         created if it doesn't already exist.
     ios_sdk (str): Where this build will be run: device or simulator.
```

Comment:
nit: to add "" to device / simulator to keep consistency with the newly added line :)

Reply:
Done!